### PR TITLE
Upgrades to 1.3.30-eap-125, and drops workarounds for KT22818.

### PR DESCRIPTION
**Pull request by rjrjr**

### CODE REVIEW

File: okio/js/src/main/kotlin/okio/ByteString.kt [hunk 1]
```diff
@@ -85,14 +86,7 @@ internal actual constructor(
    */
   actual open fun toAsciiUppercase(): ByteString = commonToAsciiUppercase()
 
-  // TODO move substring() when https://youtrack.jetbrains.com/issue/KT-22818 is fixed
```
Comment (swankjesse) [inline okio/js/src/main/kotlin/okio/ByteString.kt]: # 💥

**Comment by rjrjr** [comment]
Update rebases to keep up w/master

File: okio/jvm/src/main/java/okio/ByteString.kt [hunk 2]
```diff
@@ -141,15 +142,8 @@ internal actual constructor(
    */
   actual open fun toAsciiUppercase(): ByteString = commonToAsciiUppercase()
 
-  // TODO move substring() when https://youtrack.jetbrains.com/issue/KT-22818 is fixed
-
-  /**
```
Comment (Egorand) [inline okio/jvm/src/main/java/okio/ByteString.kt]: We should keep the KDoc in JVM module for now cause Dokka is broken. They plan to fix this in the next release, then we can move all KDoc into common.

Comment (rjrjr) [reply okio/jvm/src/main/java/okio/ByteString.kt]: Sorry, missed this comment before I merged.

**Pull request merged**